source/people-ops/benefits.html.md

```diff
@@ -8,6 +8,37 @@ meta_description: >
 All members of the team are entitled to the benefits listed in this document. If you have any doubts
 regarding any of the benefits, feel free to ask!
 
+- **[Remote-first culture](/how-we-work/where-when-and-how/#remote-work).**
+  We've been a distributed company since 2012. Every process is obsessively designed for effective
+  remote work and collaboration.
+- **[Mentoring](/personal-growth/mentoring-and-one-on-ones).**
+  When you become a member of our team, you will be assigned a mentor, who will support your
```

Comment:
You will be assigned `to` a mentor?

Reply:
I think the original sentence is actually correct... it means that you will get a mentor. What you mean on the other hand is "you are being assigned to a mentor".